resources/views/layouts/partials/header/autocomplete/categories.blade.php

```diff
@@ -3,14 +3,14 @@
         <ais-configure :hits-per-page.camel="{{ $size }}" />
     @endif
     <ais-hits v-slot="{ items }">
-        <div class="border-b p-2" v-if="items && items.length">
+        <div class="border-b py-2.5" v-if="items && items.length">
```

Comment:
All of these sub-components in the autocomplete (categories, products, all-results) all have a `py-2.5` now. Could we maybe use a gap or something like `*:py-2.5` on the parent for this instead? Seems unnecessary to give these all individual paddings?